# Author maxim-masterov (SURF)
# What: Add version to the trove-classifiers setup.py file.
# Why: EasyBuild fails at the sanitycheck step.
# Error message: "The following Python packages were likely not installed correctly because they show a version of '0.0.0'".

diff -Nru trove-classifiers-2024.4.10.orig/setup.py trove-classifiers-2024.4.10/setup.py
--- trove-classifiers-2024.4.10.orig/setup.py	2024-04-10 15:21:08.038718756 +0200
+++ trove-classifiers-2024.4.10/setup.py	2024-04-10 15:21:24.817970580 +0200
@@ -12,6 +12,7 @@
 setup(
     name="trove-classifiers",
     description="Canonical source for classifiers on PyPI (pypi.org).",
+    version="2024.4.10",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/pypa/trove-classifiers",
